### PR TITLE
Create options imageList and commonStyleString

### PR DESCRIPTION
**Pull request by suskind**
option `imageList` will be useful to declare the list of images path to be used locally as clusters images without thinking in styles. In some cases is necessary to rename the image names in the deploy process to invalidate cache and the images got a random name. 

option `commonStyleString` is used to append some extra common style to each cluster image, like opacity or other style.

### CODE REVIEW

**Comment by suskind** [comment]
I signed it! 

Thank you

File: markerclusterer/src/markerclusterer.js [hunk 1]
```diff
@@ -823,7 +855,7 @@ function Cluster(markerClusterer) {
   this.markers_ = [];
   this.bounds_ = null;
   this.clusterIcon_ = new ClusterIcon(this, markerClusterer.getStyles(),
```
Comment (stephenmcd) [inline markerclusterer/src/markerclusterer.js]: Why not have `getStyles` just include `commonStyleString_`, then no changes would be needed in `ClusterIcon`

File: markerclusterer/src/markerclusterer.js [hunk 2]
```diff
@@ -124,6 +127,18 @@ function MarkerClusterer(map, opt_markers, opt_options) {
   this.imagePath_ = options['imagePath'] ||
       this.MARKER_CLUSTER_IMAGE_PATH_;
 
+  /**
+   * @type {Array}
+   * @private
+   */
+  this.imageList_ = ('imageList' in options && options['imageList'].length) ? [].concat(options['imageList']) : null;
```
Comment (stephenmcd) [inline markerclusterer/src/markerclusterer.js]: Why not have `imageList_` get initialized here with the default 1 to 5 images if it isn't provided, then below there'd be no need for different logic based on whether `imageList_` was specified. 

Also the check should be something like `this.imageList_ = options['imageList']  ? options['imageList'] : []` which is simpler and more compatible with older browsers.

**Comment by amuramoto** [comment]
closing due to no response to comments

**Pull request closed without merging**